input.go

```diff
@@ -4,9 +4,9 @@
 
 package engi
 
-type Action int
-type Key int
-type Modifier int
+type Action 	int
+type Key 	int
+type Modifier 	int
```

Comment:
Is this according to `gofmt` or `goimports`? I feel like this will be reverted the next time we're running `gofmt` (which is all the time).